collector/mongos/server_status_test.go

```diff
@@ -77,3 +81,17 @@ func loadServerStatusFromBson(data []byte, status *ServerStatus) {
 		panic(err)
 	}
 }
+
+func TestGetServerStatusDecodesFine(t *testing.T) {
+	// setup
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Why do we need this timeout?

Reply:
1) It's [standard approach](https://github.com/mongodb/mongo-go-driver#usage) to working with the new mongodb_driver.
2) To force disconnecting from MongoDB if something goes wrong. Without it, there is a potential risk that tests can become stuck.